src/components/Navigation/Sections/FilterBar.js

```diff
@@ -10,45 +10,83 @@ export default class FilterBar extends Component {
     activeFilters: PropTypes.object,
     onFilter: PropTypes.func,
     onSort: PropTypes.func,
-    hasComponents: PropTypes.bool
+    hasComponents: PropTypes.bool,
+    showSortFilter: PropTypes.bool,
```

Comment:
not sure I am following, what other filterbar? the renderFilterBar() on PageBrowse?

I kind of prefer declaring the FilterLists that way rather than having tons of config on a single component

Reply:
yes. that is the duplicate code I was referencing. I too prefer the explicit adding of filter elements rather than one omnibus filter bar with N options. The trick is to make that other approach simple. For example, in PageBrowse, a helper function could enable the FilterList construction look like this


```js

render() {
  ...
  this.renderFilter(licenses.filter(license => license.value !== 'absence' && license.value !== 'presence'), 'License', 'license')
  this.renderFilter(providers, 'Providers', 'provider')
  this.renderFilter(curates, 'Curate', 'curate')
  ...
}

renderFilter(list, title, id) {
    return (<FilterList
          list={list}
          title={title}
          id={id}
          value={this.state.activeFilters}
          onFilter={this.onFilter}
        />
```